rts/Game/Camera/OverviewController.cpp

```diff
@@ -49,6 +49,8 @@ void COverviewController::GetState(StateMap& sm) const
 
 bool COverviewController::SetState(const StateMap& sm)
 {
-	CCameraController::SetState(sm);
+	// this camera is unique per map so we don't want to restore its settings
+	// from previous one
+	// CCameraController::SetState(sm);
```

Comment:
Doesn't this ruin `Spring.SetCameraState`? What about settings like FoV?

Reply:
> Doesn't this ruin `Spring.SetCameraState`? What about settings like FoV?

The return value of setting the camera state does not change so it should not affect any lua plugins. I can empty GetState() function as well but this is going to be just an optimization. Whether we save the state of the camera to statemap should not matter as long as we never read it. 
FoV and other properties of Overview camera are constant and are not exposed to user modification.

Reply:
I don't mean the return value, I mean actually setting these values.
```
	SetStateFloat(sm, "px", pos.x);
	SetStateFloat(sm, "py", pos.y);
	SetStateFloat(sm, "pz", pos.z);
```
Shouldn't position remain settable?

Reply:
> You should be able to do `Spring.SetCameraState({fov = 123})`, if this does not actually set FoV that's probably a bug.

thanks! checking...

Reply:
Alternatively do something like
```
Spring.Echo("fov before", Spring.GetCameraState().fov)
Spring.SetCameraState({fov = 123})
Spring.Echo("fov after", Spring.GetCameraState().fov)
```
Ideally the new value would be reflected by Get.

Reply:
Changing FOV in overview camera is not exactly working good at this moment as the height of the view will not reposition itself to fit map edges in the view resulting in edges cut off (smaller fov) or margins being too big (bigger fov). I can change it to something like that:
```
bool COverviewController::SetState(const StateMap& sm)
{
    SetStateFloat(sm, "fov", fov);
    pos.y = CGround::GetHeightAboveWater(...) // calculate zoom to fit 
    // ignore changes to direction and position
```
After the changes the camera will act smoothly, see gif: https://streamable.com/8hfk42 
Can we ignore direction or position changes to **overview camera** state? Overview camera by design should be always centered and pointing downwards but I don't know if there's any plugin out there that uses this camera in different way.

Reply:
The gif looks fine. Position XZ and dir are probably fine to ignore. I am unsure about position Y but if that simplifies a lot then it's probably ok to ignore as well.